src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+:root {
+  --link-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: block;
```

Comment:
Approve.
Just want to pay attention on this line.
It is redundant. body already has dispaly block by default
You do not need to re-request review again 🙂

Reply:
Thanks, I'll keep it in mind. I'll clean it up now